frontend/src/components/user/ATSUserLinkModal.vue

```diff
@@ -25,7 +24,9 @@ const firstName: Ref<string> = ref('');
 const lastName: Ref<string> = ref('');
 const loading: Ref<boolean> = ref(false);
 const phone: Ref<string> = ref('');
+const proponent: Ref<ATSUser | undefined> = ref(undefined);
 const selectedUser: Ref<ATSUser | undefined> = ref(undefined);
+const showCreateATS = defineModel<boolean>('showCreateATS');
```

Comment:
Not seeing a reference to this in the other changes?

Reply:
it was unused, removed, thanks